### PR TITLE
Adding info in Custom Order Status doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request (PR) describes adding missing info in Custom Order Status doc

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Custom Order Status

## Affected documentation pages

https://docs.magento.com/user-guide/sales/order-status-custom.html

## Links to Magento source code or PRs

- http://www.magedevelopers.com/blog/magento-tutorials/order-status-magento/#:~:text=%2D%20Pending%20Payment%20%2D%20This%20is%20the,will%20change%20when%20customer%20pays
- https://www.magemoto.com/blog/post/order-status

### CODE REVIEW

**Comment by leslietilling** [comment]
@dnahirna The `src/tax/tax-settings-general.md` file was updated in your previous PR -- maybe that commit got into this one by mistake?

File: src/sales/order-status-custom.md [hunk 1]
```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
...
-|Suspected Fraud|fraud||
-|Pending|pending||
-|On Hold|holded||
-|Complete|complete||
-|Closed|closed||
-|Canceled|canceled||
-|PayPal Canceled Reversal|payment_canceled_reversal||
-|PayPal Reversed|paypal_reversed||
-|Pending PayPal|pending_paypal||
+|Suspected Fraud|fraud|Sometimes orders paid via PayPal, eway or any other payment gateway, mark the orders as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:80]: eway is removed from 2.4 and deprecated in 2.3.

```suggestion
|Suspected Fraud|fraud|Sometimes orders paid via PayPal or another payment gateway are marked as Suspected Fraud. This means the order does not have invoice issued and the confirmation email is also not sent.|
```

File: src/sales/order-status-custom.md [hunk 2]
```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
...
+|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that customer has been sent to payment gateway website but no return information has been received yet. This status will change when customer pays.|
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:81]:
```suggestion
|Pending Payment|pending_payment|This is the status used if order is created and PayPal or similar payment method is used. This means that the customer was directed to the payment gateway website, but no return information has been received yet. This status will change when customer pays.|
```

File: src/sales/order-status-custom.md [hunk 3]
```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
...
+|Payment Review|payment_review|This status appear when PayPal payment review is turned on.|
+|Pending|pending|This status means no invoice and shipments have been submitted.|
+|On Hold|holded|This status can only be assigned manually. You can put any order on hold.|
+|Complete|complete|This status means that order is crated, paid and shipped to customer.|
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:85]:
```suggestion
|Complete|complete|This status means that order is crated, paid, and shipped to customer.|
```

File: src/sales/order-status-custom.md [hunk 4]
```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
...
+|Closed|closed|This status indicates that an order have had a credit memo assigned to it and the customer has received a refund.|
+|Canceled|canceled|This status is assigned manually by you or, on some payment gateways, when customer does not pay within the specified time.|
+|PayPal Canceled Reversal|payment_canceled_reversal|This status means that Paypal cancelled the reversal.|
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:88]:
```suggestion
|PayPal Canceled Reversal|payment_canceled_reversal|This status means that PayPal canceled the reversal.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

File: src/sales/order-status-custom.md (hunk 1 above)
Comment (dnahirna) [reply src/sales/order-status-custom.md:80]: Done

File: src/sales/order-status-custom.md (hunk 2 above)
Comment (dnahirna) [reply src/sales/order-status-custom.md:81]: Done

File: src/sales/order-status-custom.md (hunk 3 above)
Comment (dnahirna) [reply src/sales/order-status-custom.md:85]: Done

File: src/sales/order-status-custom.md (hunk 4 above)
Comment (dnahirna) [reply src/sales/order-status-custom.md:88]: Done

**Comment by leslietilling** [comment]
> @dnahirna The `src/tax/tax-settings-general.md` file was updated in your previous PR -- maybe that commit got into this one by mistake?

@dnahirna  Can we remove this file change from the PR? It is a merge conflict from #904.

**Comment by dnahirna** [comment]
> > @dnahirna The `src/tax/tax-settings-general.md` file was updated in your previous PR -- maybe that commit got into this one by mistake?
> 
> @dnahirna Can we remove this file change from the PR? It is a merge conflict from #904.

Done, sorry I missed this message

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

File: src/sales/order-status-custom.md [hunk 5]
```diff
@@ -77,15 +77,14 @@ A status setting cannot be unassigned from a state if the status is currently in
...
+|Payment Review|payment_review|This status appear when PayPal payment review is turned on.|
+|Pending|pending|This status means no invoice and shipments have been submitted.|
+|On Hold|holded|This status can only be assigned manually. You can put any order on hold.|
+|Complete|complete|This status means that order is crated, paid, and shipped to customer.|
+|Closed|closed|This status indicates that an order have had a credit memo assigned to it and the customer has received a refund.|
+|Canceled|canceled|This status is assigned manually by you or, on some payment gateways, when customer does not pay within the specified time.|
+|PayPal Canceled Reversal|payment_canceled_reversal|This status means that Paypal canceled the reversal.|
+|PayPal Reversed|paypal_reversed|Reversal status means that PayPal reversed the transaction.|
+|Pending PayPal|pending_paypal|This status means that order has been received by PayPal, but payment has not yet been processed.|
```
Comment (shekita88) [inline src/sales/order-status-custom.md:90]: src/images/images/config-services-oauth-authentication-locks.png

**Review by shekita88 — COMMENTED** [review]
src/images/images/config-services-oauth-authentication-locks.png

**Pull request merged**